Creating avatar and adding animations

Dennis-Nesterenko opened the pull request:
Includes
-Updated human avatar with set up humanoid rig
- Run and walk animations
- Punch, throw, and interact animations
- Breathing animation

Animations were tested and are functional in Unity

Dennis-Nesterenko committed:
Creating avatar and adding animations

Includes
-Updated human avatar with set up humanoid rig
- Run and walk animations
- Punch and interact animations

Dennis-Nesterenko committed:
Adding breathing anim

Dennis-Nesterenko committed:
Adding throw anim

stilnat commented:
There's some mismatch between the in game rigs and those animations warnings upon import, also the animations do not work on the prefab. I would advise avoiding importing animations on their own at this point, we need to fix as a whole the current issues with the prefab and the inability to animate it.

![image](https://github.com/RE-SS3D/SS3D/assets/14344825/fb3d4dcc-1ce1-46ed-acd7-d34b021e8f9b)

stilnat committed:
import animations

stilnat committed:
rename anims

stilnat committed:
create new human prefab and controller

stilnat committed:
set loop and animation names

stilnat committed:
create mask and animation scene

stilnat committed:
rename anims

stilnat committed:
update animator

stilnat committed:
crawl and sit

stilnat committed:
bake into pose

stilnat committed:
Add stuff to animation test scene

stilnat committed:
add animation rigging package

stilnat committed:
Ik punch tests

stilnat committed:
ik punching go

stilnat committed:
head tracking

Dennis-Nesterenko committed:
Adding animations

Also id go into the characters avatar go to muscle settings and enable translate DOF

Dennis-Nesterenko committed:
Merge branch 'Animations' of https://github.com/Dennis-Nesterenko/SS3D into Animations

cosmiccoincidence commented:
Once any issues are worked out, we need to make sure these make it onto the art repo as well.

stilnat commented:
closing this one as PN made a PR for my branch with the same animations, I'll make a PR including those animations and more soon.
